docs/api/en/math/Vector3.html

```diff
@@ -202,7 +202,7 @@ <h3>[method:Float dot]( [param:Vector3 v] )</h3>
 		</p>
 
 		<h3>[method:Boolean equals]( [param:Vector3 v] )</h3>
-		<p>Checks for strict equality of this vector and [page:Vector3 v].</p>
+		<p>Checks for equality of this vector and [page:Vector3 v].</p>
```

Comment:
Maybe write it like this:

    Returns *true* if this vector and [page:Vector3 v] are equal; `false` otherwise.

Reply:
Even better:

> Returns *true* if this vector and [page:Vector3 v] are strictly  equal; `false` otherwise.